WORKSPACE

```diff
@@ -367,6 +367,82 @@ rb_bundle_fetch(
         "//:rb/selenium-devtools.gemspec",
         "//:rb/selenium-webdriver.gemspec",
     ],
+    gem_checksums = {
```

Comment:
Out of curiosity, how is this block of checksums generated? Any chance we can put this in a lock file somewhere that isn't the main workspace?

Reply:
We can extract it into its own bzl file (e.g. `rb/gem_checksums.bzl`) and then `load()` it, but it won't work with Bzlmod because load's are not allowed within `MODULE.bazel`.

Ideally, this is part of `Gemfile.lock` itself and I track the work to implement it in https://github.com/bazel-contrib/rules_ruby/issues/63, but I haven't figured out how to enable checksums in Bundler yet.

This dictionary is generated by `rb_bundle_fetch()` repository rule and printed out.